package/kedro_viz/models/utils.py

```diff
@@ -0,0 +1,25 @@
+"""`kedro_viz.models.utils` contains utility functions used in the `kedro_viz.models` package"""
+
+import logging
+
+from kedro.io import AbstractDataSet
+
+logger = logging.getLogger(__name__)
+
+
+def get_dataset_type(dataset: AbstractDataSet) -> str:
+    """Get the type of a dataset as a string-- the abbreviated name of the module to
+    which ``dataset`` belongs, joined with the name of its class.
+
+    Args:
+        dataset: The dataset object to get the type of
+
+    Returns:
+        The abbreviated type of the dataset as a string.
+    """
+    if not dataset:
```

Comment:
```suggestion
    if dataset is None:
```

Not sure if this suggestion would be an improvement, but just something to thinking about... It's less safe/general but I'm not sure this will ever be called with something falsey other than `None`?

Reply:
I think this is a good point, it makes things a little less safe, but I agree with you that this is a good change.

If we ever pass anything falsey to `get_dataset_type` that isn't `None`, it would mean something upstream is broken, which we would want to know about, even if it means having an unhandled exception in a future release.

Changed in [7edd9c7](https://github.com/kedro-org/kedro-viz/pull/1232/commits/7edd9c71ab2733d37ed9639e274e2613ed1457f4).